### PR TITLE
Handling limit and offset query params added

### PR DESCRIPTION
**Pull request by Yevheniya-Deryka**
Added ability to get phones from /products endpoint with limit and offset query params for implementing pagination

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -6,25 +6,33 @@ import * as productsController from './controllers/products';
 
 const router = express.Router();
```
Comment (a-nahaitsev) [inline src/server.ts:8]: I think this router isn't already needed here

File: src/controllers/products.ts [hunk 2]
```diff
@@ -2,9 +2,35 @@ import fs from 'fs';
...
+
+    const products = JSON.parse(data.toString());
+
+    const startIndex = numbLimit * (numbOffset - 1);
+    const endIndex = numbLimit * numbOffset;
+
+    if (startIndex >= products.length) {
+
+    }
+
```
Comment (a-nahaitsev) [inline src/controllers/products.ts:30]: will this condition be used later?

**Review by a-nahaitsev — APPROVED** [review]
check some comments

File: src/server.ts (hunk 1 above)
Comment (Yevheniya-Deryka) [reply src/server.ts:8]: yep, already deleted)

File: src/controllers/products.ts (hunk 2 above)
Comment (Yevheniya-Deryka) [reply src/controllers/products.ts:30]: I'm thinking about it))

**Pull request merged**